sdk/communication/communication-common/tsconfig.json

```diff
@@ -2,7 +2,19 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "outDir": "./dist-esm",
-    "declarationDir": "./types"
+    "declarationDir": "./types",
```

Comment:
I don't think we should all of this, we already inherit these options and shouldn't define them ourselves. That's a bug in the linter if anything.

Reply:
There's an open issue
https://github.com/Azure/azure-sdk-for-js/issues/12494